public/amls.js

```diff
@@ -302,96 +302,4 @@ $(function () {
               }
         }
   }
-
-  $('[data-gov-autocomplete]').each(function() {
```

Comment:
We need to ensure all additional entries listed here exist in the new control. 👍

Reply:
I will double check this, thanks